core/src/main/scala/slamdata/interpreters/GitDataInterpreter.scala

```diff
@@ -0,0 +1,86 @@
+package precog.interpreters
+
+import cats.data.NonEmptyList
+import cats.effect.Sync
+import cats.implicits._
+import github4s.GithubResponses.GHResponse
+import github4s.domain.{Ref, RefAuthor, RefCommit, RefInfo, Tag, TreeData, TreeResult}
+import github4s.http.HttpClient
+import precog.algebras.GitData
+
+class GitDataInterpreter [F[_] : Sync](impl: github4s.algebras.GitData[F])
+                                      (implicit client: HttpClient[F], accessToken: Option[String])
+    extends GitData[F] {
```

Comment:
```suggestion
class GitDataInterpreter[F[_]: Sync](
    impl: github4s.algebras.GitData[F])(
    implicit client: HttpClient[F],
    accessToken: Option[String])
    extends GitData[F] {
```

Why is the access token `implicit`? Also the client? But really especially the access token.

Reply:
I've no idea. I just copy & pasted the declaration from github4s, taking it on faith that it made sense.

Reply:
I guess just to reduce duplication initializing all these classes on `Github`. <sigh>